app/models/remote_identity.rb

```diff
@@ -30,7 +30,8 @@
 
 class RemoteIdentity < ApplicationRecord
   belongs_to :user
-  belongs_to :oauth_client, class_name: "OAuthClient"
+  belongs_to :oauth_client, polymorphic: true
```

Comment:
I am wondering whether this should be renamed after being made polymorphic. One of the polymorphic options is for it to be an OAuth Client, so it would be confusing if the association is called `oauth_client` when it sometimes is an oauth client and sometimes not.

I'd suggest `auth_source`.

Reply:
Yes, of course, it needs to be renamed. `auth_source` works for me.